config/locales/strings/es.admin.yml

```diff
@@ -0,0 +1,34 @@
+es:
+  admin:
+    dashboard:
+      title: 'Dashboard de administración'
+      blurb: >
+        # es_ES
```

Comment:
maybe I was bitten by wrong syntax highlighting:

<img width="486" alt="Screenshot 2020-09-03 at 16 17 58" src="https://user-images.githubusercontent.com/53513496/92127035-316fa600-ee01-11ea-816b-13a1b616143f.png">

anyway I'm proud of tagging 'es' lines with the country, it's relevant information